coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/Cost.scala

```diff
@@ -3,11 +3,7 @@ package com.socrata.datacoordinator.service.collocation
 import com.rojoma.json.v3.util.{AutomaticJsonEncodeBuilder, JsonKeyStrategy, Strategy}
 
 @JsonKeyStrategy(Strategy.Underscore)
-case class Cost(moves: Int, totalSizeBytes: Long, moveSizeMaxBytes: Option[Long] = None) extends Ordered[Cost] {
-  override def compare(that: Cost): Int = {
-    this.moves - that.moves
-  }
-
+case class Cost(moves: Int, totalSizeBytes: Long, moveSizeMaxBytes: Option[Long] = None) {
```

Comment:
It might make more sense to call this something like CostVector since the actual cost is the output of the weight polynomial. As is it's a bit tricky to tell when you're referring to the computed cost vs. the feature vector.

Reply:
I guess the way I was thinking about it, is that the "cost" is the vector of different values (which actually have some meaning), and the cost is ordered. The computed weighted value is just an implementation detail of an ordering (since its value really doesn't mean anything).

Reply:
Maybe I will change my commit message to better reflect that.

Reply:
Maybe throw a comment where it responds. The reason I was confused is I've been looking at cost from that direction where it is just a number.